src/main/java/uk/gov/pay/adminusers/resources/InviteResource.java

```diff
@@ -129,6 +131,27 @@ public Invite updateInvite(@Parameter(example = "d02jddeib0lqpsir28fbskg9v0rv")
                 .collect(Collectors.toList());
         return inviteService.updateInvite(inviteCode, updateRequests);
     }
+    
+    @POST
+    @Path("/v1/api/invites/{code}/send-otp")
+    @Produces(APPLICATION_JSON)
+    public Response sendOtp(@Parameter(example = "d02jddeib0lqpsir28fbskg9v0rv") @PathParam("code") String inviteCode) {
+        LOGGER.info("Invite send OTP POST request for code - [ {} ]", inviteCode);
+
+        if (isNotBlank(inviteCode) && inviteCode.length() > MAX_LENGTH_CODE) {
+            throw new WebApplicationException(Response.Status.NOT_FOUND);
+        }
+
+        return inviteService.findInvite(inviteCode).map(inviteEntity -> {
+            if (inviteEntity.getTelephoneNumber() == null) {
+                return Response.status(PRECONDITION_FAILED).build();
```

Comment:
The error response needs to have a suitable error message with it. If we make the resource method return `void`, we would return an error response by instead throwing a `WebApplicationException`. We should add a suitable exception builder in `AdminusersExceptions` for this.